<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainQT</class>
 <widget class="QMainWindow" name="mainQT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Economie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/imgs/icon.png</normaloff>:/imgs/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="karenda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>261</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Japanese" country="Japan"/>
    </property>
   </widget>
   <widget class="QTableWidget" name="all_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>941</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>日付</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>入/出</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>内容</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>金額</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>残額</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>日別の内訳</string>
     </attribute>
     <widget class="QTableWidget" name="sel_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>日付</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>入/出</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>金額</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="sel_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>431</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu Sans'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/imgs/del.png</normaloff>:/imgs/del.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="modify_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/imgs/modify.png</normaloff>:/imgs/modify.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>内訳入力</string>
     </attribute>
     <widget class="QDateEdit" name="new_day">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Japanese" country="Japan"/>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2200</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1799</year>
        <month>12</month>
        <day>21</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy.MM.dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>日付 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>内容 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_naiyou">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/imgs/add.png</normaloff>:/imgs/add.png</iconset>
      </property>
     </widget>
     <widget class="QSpinBox" name="new_atai">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-99999999</number>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>金額 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>¥</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="csv_import_btn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/imgs/import.png</normaloff>:/imgs/import.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_out_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/imgs/out.png</normaloff>:/imgs/out.png</iconset>
    </property>
   </widget>
   <widget class="QDateEdit" name="kara">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="made">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
